.circleci/config.yml

```diff
@@ -0,0 +1,71 @@
+version: 2
+jobs:
+  build:
+    working_directory: ~/sejfguru
+    docker:
+      - image: bitwalker/alpine-elixir:1.5.3
+        environment:
+          - MIX_ENV: test
+          - DATABASE_URL: "ecto://sejfguru@localhost:5432/sejfguru_test"
```

Comment:
`ecto://` ?

Reply:
Yes, that's correct,
see https://github.com/elixir-ecto/ecto/blob/master/lib/ecto/repo.ex#L58-L65